data/json/itemgroups/vending_machines.json

```
@@ -130,5 +130,303 @@
     "subtype": "collection",
     "id": "vending_medicine",
     "entries": [ { "group": "drugs_vending", "count-min": 10, "count-max": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_adult",
+    "items": [
+      { "item": "condom", "prob": 30, "count": [ 1, 5 ] },
+      { "group": "ss_merch_1", "prob": 20 },
+      { "group": "ss_merch_2", "prob": 20 },
+      { "group": "ss_merch_3", "prob": 20 },
+      { "item": "mag_porn", "prob": 20, "count": [ 1, 3 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_useful",
+    "items": [
+      { "item": "umbrella", "prob": 10 },
+      { "item": "teleumbrella", "prob": 10 },
+      { "group": "home_hw", "prob": 50 },
+      { "group": "tools_common", "prob": 50 },
+      { "group": "cleaning", "prob": 30 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_pet",
+    "items": [
+      { "group": "petstore_shelves", "prob": 100 },
+      { "group": "petstore_misc", "prob": 100 },
+      { "group": "animalshelter_toys", "prob": 100 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_electronics",
+    "items": [
+      { "group": "consumer_electronics", "prob": 40 },
+      { "item": "light_battery_cell", "prob": 20 },
+      { "item": "medium_battery_cell", "prob": 20 },
+      { "item": "heavy_battery_cell", "prob": 10 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "vending_cars",
+    "entries": [
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "vending_bling",
+    "entries": [
+      { "group": "clothing_watch", "prob": 100 },
+      { "group": "dental_grills", "prob": 100 },
+      { "group": "accesories_personal_unisex", "prob": 100 },
+      { "group": "suits", "prob": 100 },
```

`accesories_personal_unisex` is a giant grabbag of random stuff, hence why you found relics in it. Suits being in a bling group also seems a tad weird to me.
```suggestion
      { "group": "jewelry_accessories", "prob": 100 },
      { "group": "rings_and_things", "prob": 100 },
```